README.md

```diff
@@ -22,6 +22,7 @@ A generator is included for creating web components that meet these goals.
 git clone git@github.com:patternfly/patternfly-elements.git
 cd patternfly-elements
 npm ci # install dependencies.
+npm run build # build all components
 npm run dev # spin up a local dev server
```

Comment:
If you run `npm run start` instead of `npm run dev` the build process that is needed for the dev server run.  This is definitely confusing and should be corrected in the quickstart.   `npm run build` will generate artifacts not needed just to run the dev server.   Good catch and this is something that has been needing fixed

```Suggestion
npm run start 
```